cypress/integration/granules_spec.js

```diff
@@ -167,5 +167,18 @@ describe('Dashboard Granules Page', () => {
           .next().contains('li', 'Running').contains('li', 0);
       });
     });
+
+    it('Should update the table when the Results Per Page dropdown is changed.', () => {
+      cy.visit('/granules');
+      cy.get('.filter__item').eq(3).as('page-size-input');
+      cy.get('@page-size-input').should('be.visible').click().type('10{enter}');
+      cy.url().should('include', 'limit=10');
+      cy.wait(500);
```

Comment:
Only way I could get this to work in CircleCI was to add a `wait`. Not sure if there is a better way, because otherwise it times out and fails.